src/main/scala/io/buoyant/linkerd/zipkin/KafkaInitializer.scala

```diff
@@ -19,17 +19,17 @@ case class KafkaConfig(
   initialSampleRate: Option[Double]
 ) extends TelemeterConfig {
 
-  private[this] val tracer: Tracer = {
-    val config = KafkaZipkinTracer.Config.builder()
-      .bootstrapServers(bootstrapServers.getOrElse("localhost:9092"))
-      .topic(topic.getOrElse("kafka"))
-      .initialSampleRate(initialSampleRate.map(_.toFloat).getOrElse(Sampler.DefaultSampleRate))
-      .build()
+  private[this] val config: KafkaZipkinTracer.Config = KafkaZipkinTracer.Config.builder()
+    .bootstrapServers(bootstrapServers.getOrElse("localhost:9092"))
+    .topic(topic.getOrElse("kafka"))
+    .initialSampleRate(initialSampleRate.map(_.toFloat).getOrElse(Sampler.DefaultSampleRate))
+    .build()
 
-    KafkaZipkinTracer.create(config, NullStatsReceiver)
+  def mk(params: Stack.Params): KafkaTelemeter = {
+    val param.Stats(stats) = params[param.Stats]
+    val tracer = KafkaZipkinTracer.create(config, stats.scope("zipkin.kafka"))
```

Comment:
i know we're nowhere near consistent about this, but let's call this "io.zipkin.kafka" to match the plugin name or even `configID / topic` if we think it's reasonable to run multiple telemeters to different topics